src/main/js/App.js

```diff
@@ -17,8 +17,17 @@ IconUtility.loadIconData();
 
 class App extends Component {
     componentDidMount() {
-        this.props.verifyLogin();
         this.props.getSettings();
+        this.props.verifyLogin();
+    }
+
+    componentDidUpdate(prevProps) {
+        if (this.props.systemInitialized && !prevProps.systemInitialized
+            && !this.props.logoutPerformed && !this.props.loggedIn) {
```

Comment:
could be cleaner to call the props via `const {systemInitializer, logourPerformed, loggedIn} = this.props;`